concourse/tasks/smoke-tests-run.yml

```diff
@@ -24,3 +26,8 @@ run:
       ln -snf $(pwd)/cf-release/src/smoke-tests /var/vcap/packages/smoke-tests/src/github.com/cloudfoundry/cf-smoke-tests
       set -o pipefail
       /var/vcap/jobs/smoke-tests/bin/run
+
```

Comment:
This block will not be executed if the smoke test fails. Can we do the upload in an `ensure` block?